rest_framework/validators.py

```diff
@@ -211,7 +214,9 @@ def __call__(self, attrs):
         queryset = self.exclude_current_instance(attrs, queryset)
         if queryset.exists():
             message = self.message.format(date_field=self.date_field)
-            raise ValidationError({self.field: message})
+            raise ValidationError({
+                self.field: message,
+            })
```

Comment:
What I mean is, whatever we do to expose it later on, this structure looks wrong:
```
{
    'field': 'error', 
    'code': 'the code'
}
```
while this looks better
```
{
    'field': {
        'message': 'error',
        'code': 'the code'
    }
}
```